src/main/java/medicaldoctor/controllers/ProfileController.java

```diff
@@ -35,14 +34,27 @@ void change_password(ActionEvent event) {
     
     @Override
     public void initialize(URL location, ResourceBundle resources) {
-        first_name_tf.setText(AppSession.getActiveUser().getFirstName());
-        last_name_tf.setText(AppSession.getActiveUser().getLastName());
-        username_tf.setText(AppSession.getActiveUser().getUserName());
-        employee_id_t.setText(AppSession.getActiveUser().getId() + "");
-        email_tf.setText(AppSession.getActiveUser().getEmail());
-        department_tf.setText(AppSession.getActiveUser().getDepartment());
-        office_tf.setText(AppSession.getActiveUser().getOfficeNum() + "");
-        employee_type_tf.setText(AppSession.getActiveUser().getType().getName());
+        if(AppSession.getAdminFlag() == 1){
+            first_name_tf.setText(AppSession.getAdminUserSelection().getFirstName());
```

Comment:
why is there two user selections

Reply:
The admin can view different profiles. That's why there's a flag. If the admin clicked it the flag would be raised and allow a different user to be loaded.

Reply:
that makes sense